models/webhooks/store_modifier_updated.yml

```diff
@@ -0,0 +1,36 @@
+type: object
+properties:
+  store/modifier/updated:
+    description: |
+      Fires when you edit attributes for a product modifier globally or for a channel locale (i.e. overrides). The `context` fields are present for only override updates. 
+      
+      For shared modifiers, the maximum number of product IDs in a payload is 100. If a shared modifier is assigned to more than 100 products, the payload includes a link to the API so you can query all the event data.  
+    type: object
+    properties:
+      scope:
+        type: string
+      store_id:
+        type: string
+      data:
+        type: object
+        properties:
+          type:
+            type: string
+          id:
+            type: integer
+          affected_product_ids: 
```

Comment:
Should this be `product_ids` , as in the [google doc](https://docs.google.com/document/d/1uTeDIkd33EtmQrshLtp0Yh3Qows6OFj5PSZvNmP8n0k/edit)?